app/controllers/services_controller.rb

```diff
@@ -12,5 +13,6 @@ def register
     render json: service,
            only: :url,
            include: { models: { only: :name } }
+    if service.changed? then MicroservicesRouter.notify_services_of_changes end
```

Comment:
Does this handle changes to dependent objects, i.e. the service is the same but is now responsible for changed models?